mesheryctl/pkg/utils/helpers.go

```diff
@@ -45,23 +45,38 @@ const (
 	MeshsyncURL   = baseConfigURL + "samples/meshery_v1alpha1_meshsync.yaml"
 
 	// Documentation URLs
-	docsBaseURL    = "https://docs.meshery.io/"
-	rootUsageURL   = docsBaseURL + "reference/mesheryctl"
-	perfUsageURL   = docsBaseURL + "reference/mesheryctl/perf"
-	systemUsageURL = docsBaseURL + "reference/mesheryctl/system"
-	meshUsageURL   = docsBaseURL + "reference/mesheryctl/mesh"
-
+	docsBaseURL      = "https://docs.meshery.io/"
+	rootUsageURL     = docsBaseURL + "reference/mesheryctl"
+	perfUsageURL     = docsBaseURL + "reference/mesheryctl/perf"
+	systemUsageURL   = docsBaseURL + "reference/mesheryctl/system"
+	meshUsageURL     = docsBaseURL + "reference/mesheryctl/mesh"
+	expUsageURL      = docsBaseURL + "reference/mesheryctl/exp"
+	filterUsageURL   = docsBaseURL + "reference/mesheryctl/exp/filter"
+	patternUsageURL  = docsBaseURL + "reference/mesheryctl/pattern"
+	appUsageURL      = docsBaseURL + "reference/mesheryctl/app"
+	contextDeleteURL = docsBaseURL + "reference/mesheryctl/system/context/delete"
+	contextViewURL   = docsBaseURL + "reference/mesheryctl/system/context/view"
+	contextCreateURL = docsBaseURL + "reference/mesheryctl/system/context/create"
+	contextUsageURL  = docsBaseURL + "reference/mesheryctl/system/context"
```

Comment:
For ```system channel``` we can handle but I have logged out most of information to the user.

Reply:
IMO, we should still do it for sake of consistency across the codebase.

Reply:
@alphaX86 fyi

Reply:
There would be some folks who would search for links... I know it would be a bit overkill but as @nishantwrp said we need to be consistent on the codebase, if you include links, then include for all of them

Reply:
Let's then do it :+1: